src/applications/claims-status/components/claim-files-tab/FilesOptional.jsx

```diff
@@ -19,13 +19,25 @@ function FilesOptional({ item }) {
           ? item.friendlyName
           : item.displayName}
       </h4>
-      <p>Requested to others on {dateFormatter(item.requestedDate)}</p>
+      <p>
+        {cstFriendlyEvidenceRequests
+          ? `Requested from outside VA on ${dateFormatter(item.requestedDate)}`
+          : `Requested to others on {dateFormatter(item.requestedDate)}`}
```

Comment:
This is minor, and I don't know if you have authority to change it (i.e. it might be in official design docs) but grammatically this should be "Requested **of** others on ...," not "Requested **to** others on ..."